importer_client/python/tools/timesketch_importer.py

```diff
@@ -462,6 +468,19 @@ def main(args=None):
         help=("Path to the file that is to be imported."),
     )
 
+    config_group.add_argument(
+        "--analyzer_names",
+        "--analyzer-names",
+        type=str,
+        action="store",
+        dest="analyzer_names",
+        default=[],
+        help=(
+            "Set of analyzers that we will automatically run right after the"
+            "timelines are uploaded. The input needs to be the analyzers names."
```

Comment:
Please explain what is expected as input. A List, a Set, a comma separated string?